### DISCUSSION TITLE
도적질 카드들의 문제

### DISCUSSION THREAD

**Discussion by 115dkk** [scarf005/Marisa#120] [Balance]
https://m.dcinside.com/board/slay/119053?headid=&recommend=&s_type=subject_m&serval=%EB%A7%88%EB%A6%AC%EC%82%AC

아주 단순함

저승천에서는 스톨링을 통해 치트키가 되고

반대로 고승천에서는 방어 카드가 뜨는 걸 방해하는 원망스러운 유사타격 또는 저주(욕심쟁이 마녀)임

이거 그냥 '마리사는 엄청난 걸 훔쳐갔습니다' 딱 하나에서 온 거 아님?

해당 카드풀을 배제하는 옵션이 모드에 있는지 모르겠는데, 있다면 그 옵션을 추가해주면 좋겠음

그리고 이게 도적질이 맞는지는 모르겠지만, 일시적으로 유물 하나 먹는 카드와 유물 개수를 카운트해서 딜하는 카드도 이게 맞나 모르겠음. 특히 유물 개수 카운트 딜카드는 ㄹㅇ 강약약강 느낌 많이 나

**Comment by scarf005** [comment]
도적질 컨셉 자체는 원판에서 많이 나오긴 했음
문제는 장수가 너무 적어서 자기만의 테마는 만들 수 없고 다른 카드랑 시너지가 없어서 생기는 것 같음

도둑질, 욕심쟁이 마녀 -> 골드주는 타격, 연금
골동품, 보물 사냥꾼 -> 유물

1. 골드, 유물 테마를 키워서 카드 시너지를 늘리기 -> 이미 테마가 많음
2. 다른 테마와 시너지 더 주기 -> 이미 카드 효과가 복잡함
3. 전부 지워버리기 -> 카드 일러스트가 아까움
4. 골드와 유물 관련 효과 카드들를 유물로 대체하기 -> 가장 현실적일 것 같은데 시간이 좀 걸림

일단은 의견대로 설정에서 골드 컨셉 카드와 유물 컨셉 카드를 뺄 수 있도록 해보겠음

**Reply by 115dkk** [reply]
그와 별도로, 도둑질 컨셉에 나는 한가지 더 문제가 있다고 생각함

이런 일종의 스톨링 요소들을 뜯어보면

포식 - 솔직히 밸류 낮은 편인 최체를 올려줌. 많이 처먹으면 체감이 좀 되지만, 정말 작정하고 처먹는게 아닌 이상 골드나 유물마냥 극적인 효과는 절대 아님
연금 - 포션은 최대 제한이 있어서 강해지는 것에 한계가 있음
유전 알고리즘 - 자신의 깡방어도만 늘림
교훈 - 강화는 역시 카드 숫자만큼만 가능한 한계가 있음. 심지어 와쳐는 압축 지향 덱이 워낙 쎄서 더더욱 그 약빨이 떨어지는 감이 있음
탐손 - 얻기 무지하게 어려운 무색 카드

그에 비해 마리사가 도둑질로 얻는 밸류는 주로 골드, 유물인데 얘네들은 위에 써놓은 것에 비해 훨씬 범용성과 잠재력이 높음

그래서 저승천에서 치트키같은 요소가 되는거라고 생각함

**Reply by scarf005** [reply]
일단은 더 나은 방법을 찾기 전까지는 #125 에서처럼 설정에서 카드 등장을 막는 옵션을 추가해보겠음

**Comment by erasels** [comment]
One thing to think about if you do go through with removing them is that you'd be reducing the number of cards in the card pool which leads to runs feeling more similar. Even with 5 cards less it can be noticeable.

Furthermore, not every card in a character should be a synergy card, you need decent glue cards or fillers, otherwise you harm the drafting choices of players since you don't really pick cards from anohter archteype once you've committed to one.
